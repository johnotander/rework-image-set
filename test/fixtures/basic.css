.bg-img {
  background-image: image-set('my-img.png' 1x,
                              "my-img-2x.png" 2x,
                              'my-img-print.png' 600dpi);
}

.some-other-class {
  color: #fafafa;
}
