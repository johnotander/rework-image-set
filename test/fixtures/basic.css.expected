.bg-img {
  background-image: url('my-img.png');
  background-image: -webkit-image-set('my-img.png' 1x,
                                      'my-img-2x.png' 2x,
                                      'my-img-print.png' 600dpi);
}
